provider/kubernetes/kubernetes.go

```diff
@@ -381,10 +382,18 @@ func getRuleForPath(pa v1beta1.HTTPIngressPath, i *v1beta1.Ingress) string {
 
 	rules := []string{ruleType + ":" + pa.Path}
 
+	rewrited := false
 	if rewriteTarget := i.Annotations[annotationKubernetesRewriteTarget]; rewriteTarget != "" {
 		rules = append(rules, ruleTypeReplacePath+":"+rewriteTarget)
+		rewrited = true
```

Comment:
I just realized that we also have the case where the rule is explicitly set to `ReplacePath`. We can either handle that in this PR too or do a follow up.